docs/Custom-Brimcap-Config.md

```diff
@@ -102,11 +106,11 @@ logs and open flows from the pcap via the **Packets** button.
 The same combination of `brimcap` and `zed` commands can be used to
 incrementally add more logs to the same pool and index for additional pcaps.
 
-The setting in the Zui **Settings** for the **Brimcap YAML Config File**
+The Zui **Settings** for the **Brimcap YAML Config File**
 can also be pointed at the path to this configuration file, which will cause it
 to be invoked when you open or drag pcap files into Zui.
 
-![Zui YAML Config File Preference](media/Zui-Pref-YAML-Config-File.png)
+![Zui YAML Config File Setting](media/Zui-Settings-YAML-Config-File.png)
```

Comment:
The Zui menu used to be called "Preferences" in some OSes and "Settings" on others, but thankfully we've standardized on "Settings" across the board now.